Makefile

```diff
@@ -315,12 +315,17 @@ codespell:
 .PHONY: validate
 validate: lint .gitvalidation validate.completions man-page-check swagger-check tests-expect-exit pr-removes-fixed-skips
 
+
+# The image used below is generated manually from contrib/validatepr/Containerfile in this podman repo.  The builds are
+# not automated right now.  The hope is that eventually the quay.io/podman:stable image is multiarch and can replace this
```

Comment:
Do you mean the quay.io/libpod/fedora_podman?
quay.io/podman/stable is the official podman image and does not contain development tools AFAIK